svthalia/Reaxit
# Fix/registration members limited to 50

Closes #146 

### Summary
Added lazy loading (pagination) of the event registrations in the event screen and a high limit for the admin registrations.

### How to test
Steps to test the changes you made:
1. Log into the app.
2. Open an event with more than 50 members
3. Observe how the entire list of registrations will be loaded.
